.docker/Dockerfile

```diff
@@ -1,9 +1,35 @@
-FROM jekyll/jekyll:4.0.0
+FROM ruby:2.7.1
 
-RUN apk add --no-cache --no-progress\
-    graphviz\
-    openjdk8-jre-base\
-    fontconfig\
-    ttf-dejavu
+WORKDIR /srv/jekyll
+COPY . .
 
-RUN gem install bundler
+# Ruby env
+ENV BUNDLE_APP_CONFIG=/usr/local/bundle
+ENV BUNDLE_BIN=/usr/local/bundle/bin
+ENV GEM_BIN=/usr/gem/bin
+ENV GEM_HOME=/usr/gem
+
+
+RUN addgroup jekyll && \
+    useradd -g jekyll jekyll
+
+RUN mkdir -p /var/jekyll
+RUN mkdir -p /srv/jekyll
+RUN chown -R jekyll:jekyll /srv/jekyll
+RUN chown -R jekyll:jekyll /var/jekyll
+RUN chown -R jekyll:jekyll /usr/local/bundle
+
+RUN apt-get update && \
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5414481)